README.md

```diff
@@ -119,4 +119,45 @@ module "transfer" {
   bucket_name                = aws_s3_bucket.transfer-bucket.id
   bucket_arn                 = aws_s3_bucket.transfer-bucket.arn
 }
+```
+
+## OpenVPN
+
+```
+module "vpn" {
+  source         = "git@github.com:in4it/terraform-modules.git//modules/openvpn"
+  aws_region     = var.aws_region
+  aws_account_id = var.aws_account_id
+  env            = var.env
+  domain         = var.domain
+  project_name   = "my_client"
+
+  vpc_id          = data.terraform_remote_state.network.outputs.vpc_id
```

Comment:
maybe you could also add the terraform vpc module here to give a better example of what we're looking for